run/logging-manual/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Google LLC. All rights reserved.
```

Comment:
This file is copied verbatim from the quickstart. Longer term I'm hoping to create a template mechanism so default Dockerfiles in the samples can be maintained in one place.